packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -78,6 +80,12 @@ export function setSpecJSONHandler(writer: (json: string) => void): void {
 function getTestRunner(tags, persistenceEnabled): Function {
   if (tags.indexOf(NO_WEB_TAG) >= 0) {
     return it.skip;
+  } else if (
+    !persistenceEnabled &&
+    tags.indexOf(NO_MEMORY_PERSISTENCE) !== -1
```

Comment:
We used to have a tag that was called just 'PERSISTENCE', which might imply that it is durable.

Reply:
I agree with the general principle, but I did it this way to match how the rest of the tags were implemented. Do you think it's worth deviating?

Reply:
benchmark, exclusive, and multiclient follow the positive form. I think we should continue that.

no-web, no-ios, and no-android are a mistake; I think they should have been phrased the other way too, especially since we tend to add a feature on a new platform, so it's easier to phrase this in terms of which platform is going first.

That leaves no-lru, which I think we could stand to be positive, but I don't know what the positive form would be.

In any case, I think positive phrasing could be a long term goal and this would be a step in that direction.

Reply:
Switched the tags to be positive.

Reply:
Switch the platform tags back to negative.